### PR TITLE
Adding pid & lyap_control pkgs for kinetic.

### PR DESCRIPTION
**Pull request by AndyZe**

### CODE REVIEW

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -273,6 +273,21 @@ repositories:
...
       version: hydro-devel
     status: maintained
+  lyap_control:
+    doc:
+      type: git
+      url: https://bitbucket.org/AndyZe/lyap_control.git
+      version: master
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
```
Comment (dirk-thomas) [inline kinetic/distribution.yaml]: You can't just use the Indigo release. You need to run bloom for the Kinetic for your repo.

**Comment by dirk-thomas** [comment]
Yoy cannot just rename the release tags. You **need** to run a bloom release for Kinetic.

**Comment by AndyZelenak** [comment]
@dirk-thomas OK. I interpreted from the email to ros-users that rosdistro was the first step:
"Once your package has been added to Kinetic in rosdistro, you can generate
a prerelease command on prerelease.ros.org."

**Comment by dirk-thomas** [comment]
If you want to adda release entry to do a prerelease it should not have a version field.

**Comment by wjwwood** [comment]
I believe you just need the source entry so you can run a prerelease (using the devel option). You can omit the release entry until your first release I think.

**Pull request closed without merging**